pkgs/development/libraries/qhull/default.nix

```diff
@@ -1,13 +1,14 @@
 { lib, stdenv, fetchFromGitHub, cmake }:
 
-stdenv.mkDerivation {
-  name = "qhull-2016.1";
+stdenv.mkDerivation rec {
+  pname = "qhull";
+  version = "8.0.2";
```

Comment:
It seems that the stable release versioning scheme is year-based. Could you change the version to `2020.2`, with `rev = version;`?

Reply:
the repo tagged the release with both and mention both in changelog http://www.qhull.org/src/Changes.txt. I changed it as requested

Reply:
Judging by the phrase "Qhull 8.0.2 (2020.2 candidate)" on their github, the fact that 2020.2 was tagged a week after v8.0.2 (at the same commit), and that many v releases do not have a year release it seems that v releases are like release candidates and yearly releases are stable.

Please fix the commit title, currently it says "1016.1" instead of "2016.1".

Reply:
sry done.